remedy.lic

```diff
@@ -140,10 +142,16 @@ class Remedy
       command = "turn my #{@container} with my pestle"
     when 'Patiently you wait a few moments and then proceed'
       waitrt?
-      crush("crush my #{@noun} with my pestle")
+      command = "crush my #{@noun} with my pestle"
     when 'Interesting thought really... but no.'
       finish
     when 'With short strokes you crush'
+      waitrt?
+      command = "crush my #{@noun} with my pestle"
+    when 'Roundtime:'
+      waitrt?
+      command = "crush my #{@noun} with my pestle"
+    when 'To avoid a problem'
       case bput("analyze my #{@noun}",
```

Comment:
These nested when/case blocks are really hard to read. Could you extract the inner one into its own function?